AddOns/BasicFinancials/app/src/page/AssistedSetupBF.Page.al

```diff
@@ -0,0 +1,154 @@
+page 20600 "Assisted Setup BF"
+{
+    Caption = 'Basic Assisted Setup';
+    DeleteAllowed = false;
+    InsertAllowed = false;
+    LinksAllowed = false;
+    PageType = NavigatePage;
+    ShowFilter = false;
+
+    layout
+    {
+        area(content)
+        {
+            label(Title)
+            {
+                Caption = 'Welcome';
+                Style = StandardAccent;
+                ApplicationArea = All;
+            }
+            group(Instructional)
+            {
+                Caption = ' ';
+                InstructionalText = 'The Basic extension enables the subset Dynamics 365 Business Central capabilities provided by the Basic license.';
+            }
+            field(DocumentationPart; 'For more information, see the documentation.')
+            {
+                ApplicationArea = Basic, Suite;
+                Caption = ' ';
+                ShowCaption = false;
+                Editable = false;
+                ToolTip = 'For more information, see the documentation.';
+            }
+            field(HelpLink; 'Basic documentation')
+            {
+                ApplicationArea = Basic, Suite;
+                Caption = ' ';
+                ShowCaption = false;
+                Editable = false;
+                ToolTip = 'Basic documentation';
+
+                trigger OnDrillDown()
+                begin
+                    Hyperlink('https://go.microsoft.com/fwlink/?linkid=');
+                end;
+            }
+            field(ConsentPart; 'To enable the service you must read and accept the terms of use.')
+            {
+                ApplicationArea = Basic, Suite;
+                Caption = ' ';
+                ShowCaption = false;
+                Editable = false;
+                ToolTip = 'By enabling this extension you accept the terms of use. To enable the service you must read and accept the terms of use.';
+            }
+            field(TermsOfUseLink; 'Basic terms of use')
+            {
+                ApplicationArea = Basic, Suite;
+                Caption = ' ';
+                ShowCaption = false;
+                Editable = false;
+                ToolTip = 'Basic terms of use';
+
+                trigger OnDrillDown()
+                begin
+                    Hyperlink('https://go.microsoft.com/fwlink/?linkid=');
+                end;
+            }
+            field(AcceptConsent; ConsentAccepted)
+            {
+                ApplicationArea = Basic, Suite;
+                Editable = true;
+                Caption = 'I understand and accept the terms';
+                ToolTip = 'Acknowledge that you have read and accept the terms.';
+            }
+            /* Temporarily removed due to issue regarding License check
+            field(IsSupportedLicenses; IsSupportedLicenses)
```

Comment:
Please replace IsSupportedLicenses and IsSupportedLicense to HasBCBasicLicense ?